### PR TITLE
195 preview

### PR DESCRIPTION
**Pull request by tmmi**
Preview raw content from messages published on frontend in management. 
Endpoints:
 /topics/{topicName}/preview -> return list of preview messages content as JSON
 /topics/{topicName}/preview/{idx} -> return one preview message as octet-stream

### CODE REVIEW

File: hermes-common/src/main/java/pl/allegro/tech/hermes/infrastructure/zookeeper/ZookeeperPaths.java [hunk 1]
```diff
@@ -118,4 +119,8 @@ public String subscriptionMetricPathWithoutBasePath(TopicName topicName, String
                 METRICS_PATH,
                 metricName);
     }
+
+    public static String previewElementPath(String previewPath, String i) {
```
Comment (pawelszymczyk) [inline hermes-common/src/main/java/pl/allegro/tech/hermes/infrastructure/zookeeper/ZookeeperPaths.java]: What is 'i'?

**Comment by druminski** [comment]
:+1:

File: hermes-common/src/main/java/pl/allegro/tech/hermes/infrastructure/zookeeper/ZookeeperPaths.java (hunk 1 above)
Comment (tmmi) [inline hermes-common/src/main/java/pl/allegro/tech/hermes/infrastructure/zookeeper/ZookeeperPaths.java]: index of preview element, I will change parameter name.

**Comment by pawelszymczyk** [comment]
:+1:

**Comment by adamdubiel** [comment]
This code is not thread safe and leads to bugs - this merge will be reverted and code rewritten (EvictingQueue is not thread safe).

**Pull request merged**